app/Http/Controllers/Widgets/TopInterfacesController.php

```diff
@@ -54,7 +54,8 @@ public function getView(Request $request)
             ->isValid()
             ->select('port_id', 'device_id', 'ifName', 'ifDescr', 'ifAlias')
             ->groupBy('port_id', 'device_id', 'ifName', 'ifDescr', 'ifAlias')
-            ->where('poll_time', '>', Carbon::now()->subMinutes($data['time_interval'])->timestamp)
+            ->where([['poll_time', '>', Carbon::now()->subMinutes($data['time_interval'])->timestamp],
+                     ['ifOperStatus', '=', 'up']])
```

Comment:
Maybe isUp() instead maybe?

Reply:
indeed ... i always forget the macros we have :D